packages/core/graph/src/Graph.js

```diff
@@ -479,6 +472,16 @@ export default class Graph<TNode, TEdgeType: number = 1> {
     return null;
   }
 
+  topoSort(): Array<NodeId> {
```

Comment:
We need call topological sort on our asyncBundleRootGraph and reverse the result for 1 main reason, which was
we needed to process all of a node's **parents** before visiting itself. 

See the topoSort() call in `createIdealGraph()` for more information on this.